### PR TITLE
Fix arm arch previously only set for Android

### PR DESCRIPTION
**Pull request by simonblee**
Specify library name and version:  **lib/1.0**

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/wiki#how-to-submit-a-pull-request) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/openh264/all/conanfile.py [hunk 1]
```diff
@@ -44,17 +44,17 @@ def build(self):
...
+                elif self.settings.arch == 'x86_64':
+                    arch = 'x86_64'
                 else:
-                    if self.settings.arch == 'x86':
-                        arch = 'i386'
-                    elif self.settings.arch == 'x86_64':
-                        arch = 'x86_64'
-                    else:
-                        arch = self.settings.arch
+                    arch = self.settings.arch
```
Comment (madebr) [inline recipes/openh264/all/conanfile.py]:
```suggestion
                arch = {
                    "armv7": "arm",
                    "armv8": "arm64",
                    "x86": "i386",
                }.get(str(self.settings.arch), str(self.settings.arch))
```

Comment (simonblee) [reply recipes/openh264/all/conanfile.py]: That's cleaner

**Comment by danimtb** [comment]
@simonblee in order to move this PR forward, please request access in #4

**Comment by danimtb** [comment]
Closing this one due to inactivity

**Pull request closed without merging**